plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/discoverer/XcpServerDiscoverer.java

```diff
@@ -529,13 +540,12 @@ public boolean processCommands(long agentId, long seq, Command[] commands) {
     }
 
     private void createXsToolsISO() {
-        String isoName = "xs-tools.iso";
-        VMTemplateVO tmplt = _tmpltDao.findByTemplateName(isoName);
+        VMTemplateVO tmplt = _tmpltDao.findByTemplateName(_toolsIsoName);
         Long id;
         if (tmplt == null) {
             id = _tmpltDao.getNextInSequence(Long.class, "id");
             VMTemplateVO template =
-                    VMTemplateVO.createPreHostIso(id, isoName, isoName, ImageFormat.ISO, true, true, TemplateType.PERHOST, null, null, true, 64, Account.ACCOUNT_ID_SYSTEM,
+                    VMTemplateVO.createPreHostIso(id, _toolsIsoName, _toolsIsoName, ImageFormat.ISO, true, true, TemplateType.PERHOST, null, null, true, 64, Account.ACCOUNT_ID_SYSTEM,
```

Comment:
I do not know what you think about the use of "_" here to denote object attributes, but I do not see any benefit on using it. As a matter of fact, I see most PRs removing them.

Reply:
I, honestly, don't like it and have never used it like this, but wanted to follow the same pattern in the code base.

Reply:
It is always good to create a new trend :)